package.json

```diff
@@ -35,9 +35,11 @@
     "react-dom": "^0.14.x "
   },
   "devDependencies": {
-    "babel": "^5.x.x",
-    "babel-core": "^5.x.x",
-    "babel-loader": "^5.x.x",
+    "babel": "~6.0.15",
+    "babel-core": "~6.0.15",
+    "babel-loader": "~6.0.1",
+    "babel-preset-es2015": "~6.0.15",
+    "babel-preset-react": "~6.0.15",
```

Comment:
:+1: Copypasta issue, happy to have it as existing - although would advise on: http://stackoverflow.com/questions/22343224/difference-between-tilde-and-caret-in-package-json

Ideally there should be no breaking changes between say 6.0 -> 6.1, but not always guaranteed